src/app/workflow/tabs/manageVariables.html

```
@@ -0,0 +1,32 @@
+<section>
+  <form class="form-horizontal with-padding" role="form">
+    <fieldset>
+      <legend>
+        Update state variable
+      </legend>
+      <div class="form-group">
+        <label class="col-md-3 control-label" for="variableName">Name</label>
+        <div class="col-md-9">
+          <input class="form-control" id="variableName" ng-model="ctrl.model.variableName"/>
+        </div>
+      </div>
+      <div class="form-group">
+        <label class="col-md-3 control-label" for="variableValue">Value</label>
+        <div class="col-md-9">
+          <textarea class="form-control" id="variableValue" ng-model="ctrl.model.variableValue"/>
```

Must be:
```
<textarea ...></textarea>
```
Otherwise rest of the form will not be rendered (fails silently).